tests/conftest.py

```diff
@@ -205,6 +207,24 @@ def user(db, django_user_model, django_username_field):
     return user
 
 
+def create_authenticated_client(client, user):
```

Comment:
fixtures can AFAIR also be called like normal functions, so why not just inline this in `authenticated_client` below?

Reply:
Nope, not anymore. That behaviour changed in pytest versions, don't know where exactly; that's actually the only reason I changed this.